modules/security-context-constraints-about.adoc

```diff
@@ -109,12 +110,12 @@ The `privileged` SCC allows:
 
 [NOTE]
 ====
-Setting `privileged: true` in the pod specification does not select the `privileged` SCC. Setting `privileged: true` in the pod specification matches on the `allowPrivilegedContainer` field of an SCC.
+Setting `privileged: true` in the pod specification does not select the `privileged` SCC. However, setting `privileged: true` in the pod specification considers only those SCCs that have a match on their `allowPrivilegedContainer` field, of which the `privileged` SCC is one.
```

Comment:
In my opinion, mentioning ``However, setting `privileged: true`` is necessary because it's a key-value pair and also user may get confused if we mention this in singular third person pronoun. 

Also if you see the previous version of the documentation, it starts with ``setting `privileged: true``.
![image](https://user-images.githubusercontent.com/87811193/145963062-e0d191f0-c131-4378-a6cc-0d1646b2a09b.png)

Let me know your thoughts on this.

Reply:
"However" doesn't seem to justify the importance of the words that follow,  if that makes any sense, could be considered just a nit.

Reply:
Instead of nit picking this wording, we can rephrase it entirely to see if it reads more clearly.

For example:

Setting privileged: true in the pod specification does not necessarily select the privileged SCC.  The most appropriate SCC (of which the privileged SCC is one by default) which has "allowPrivilegedContainer: true" will be chosen if the user has permission to use it.

Reply:
> Instead of nit picking this wording, we can rephrase it entirely to see if it reads more clearly.
> 
> For example:
> 
> Setting privileged: true in the pod specification does not necessarily select the privileged SCC. The most appropriate SCC (of which the privileged SCC is one by default) which has "allowPrivilegedContainer: true" will be chosen if the user has permission to use it.

👍 Just a minor edit

Setting `privileged: true` in the pod specification does not necessarily select the `privileged` SCC. The most appropriate SCC containing "allowPrivilegedContainer: true" will be selected if the user has permission to use it.

Reply:
Yes!  Your reword of my rephrase is ALMOST perfect, and very close to my original intent.

The problem is that if you take a regular workload and just throw "privileged: true" at it, you're actually (from experience) more likely to get admitted through the "node-exporter" SCC, and not privilged SCC at all.

Reply:
Agreed, how about

Setting `privileged: true` in the pod specification does not necessarily select the `privileged` SCC. The most appropriate SCC containing `"allowPrivilegedContainer: true"` (out of which the `privileged` SCC is one by default) will be selected if the user has permission to use it.

Reply:
@y4sht We usually avoid putting content in parenthesis because users tend to ignore the content in parenthesis. There are exceptions like when expanding abbreviations, SI units.

Reply:
@xingxingxia suggested
Setting `privileged: true` in the pod specification does not necessarily select the `privileged` SCC. The most appropriate SCC which has `"allowPrivilegedContainer: true"` will be chosen if the user has permission to use it. The prioritization section of this documentation determines which one is the most appropriate.

Reply:
I think this phrasing is great.

Reply:
@l4zy0n3 
I didn't understand this:
`The prioritization section of this documentation determines which one is the most appropriate. ?`
Is this an explanation or you want to include this as well ?

Reply:
Yes, it would be great if we could add this explaination, it makes the phrasing much clearer.

Reply:
@y4sht The prioritization section of this documentation - - which section you're referring to ? I wanna give cross-reference to that section.

Reply:
It's here https://deploy-preview-39717--osdocs.netlify.app/openshift-enterprise/latest/authentication/managing-security-context-constraints.html#scc-prioritization_configuring-internal-oauth

Reply:
(and it has a REALLY weird anchor tag name.  Should just be #scc-prioritization I think, but if it's being linked to from elsewhere we don't want to break those links)

Reply:
@dbaker-rh i feel it's better not to touch anchor tags. As you mentioned it might break if it's reference somewhere else.

Reply:
@y4sht @dbaker-rh @xingxingxia I can't cross reference **Security context constraints prioritization** section because it's a part of  **About security context constraints** module. In our doc repo we have defined rule that we can't reference reference module in a module.

Let me give a more context on why we can't cross-reference module in a module -- explained by one of our DPM .
Having xref in a module, it breaks the contract of a module. It is no longer a module if it taking you to somewhere else, and that was one of the reasons this rule was put into place.

Reply:
@y4sht @dbaker-rh expect cross reference, I have made the other changes. Please let me know if there's any other way we can put this up.

Reply:
@y4sht PTAL and let me know your feedback.

Reply:
@darshan-nagaraj what about
The SCC which has `allowPrivilegedContainer: true` and has the highest prioritization will be chosen if the user has the permission to use it.

Reply:
@y4sht Yes, makes sense. I will make the changes.

Reply:
@y4sht Changes updated. PTAL and if it's okay for you give me QA ack. Thanks